pkgs/applications/audio/roomeqwizard/default.nix

```diff
@@ -69,13 +71,17 @@ stdenv.mkDerivation rec {
     runHook postBuild
   '';
 
+  FONTCONFIG_PATH="${fontconfig.out}/etc/fonts/"; # resolves a buildtime error, doesn't improve the fonts
+
+  # GTK_PATH wanted at runtime (looks for adwaita), doesn't improve looks either
   installPhase = ''
     runHook preInstall
 
     mkdir -p $out/bin $out/lib/udev/rules.d $out/share/icons/hicolor/256x256/apps
     makeWrapper $out/share/roomeqwizard/roomeqwizard $out/bin/roomeqwizard \
       --set INSTALL4J_JAVA_HOME_OVERRIDE ${jdk8} \
-      --prefix PATH : ${lib.makeBinPath [ coreutils gnused gawk ]}
+      --prefix PATH : ${lib.makeBinPath [ coreutils gnused gawk ]} \
+      --prefix GTK_PATH : "${gnome-themes-extra}/lib/gtk-2.0"
```

Comment:
If it doesn't improve the look & feel, should it be left here? This makes this package depend on `gnome-themes-extra` for no clear benefit.

Reply:
i consider getting rid of a runtime warning a clear benefit
but think that someone with more insight in that warning (or the buildtime error) may think otherwise
which is why i've included those changes as separate commits

Reply:
This is the closure size on your first commit (update):
```
/nix/store/p8ygbc0chc6c9a96p02a1f6dii01qsr5-roomeqwizard-5.20.13	252.3M
```

on second (+fontconfig):
```
/nix/store/x36f1qmj5lz5d6bd10lg2c3pxd4xka8g-roomeqwizard-5.20.13	252.3M
```

on third (update+fontconfig+gtk):
```
/nix/store/5k7q2lk53jy037sq20ndn2ndckcjfirn-roomeqwizard-5.20.13	352.3M
```

I find it debatable to have 100M more in closure size to solve a warning. Especially if you don't care about the theme (not using Gnome and even more on MacOS).

Reply:
fair point, removing the "cosmetic" commits (that don't affect the cosmetics xD)